content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -400,6 +400,7 @@ behavior:
 ```
 
 `periodSeconds` indicates the length of time in the past for which the policy must hold true.
+The value of `periodSeconds` must be less than or equal to 1800 (half an hour).
```

Comment:
```suggestion
The default maximum value of `periodSeconds` is 1800 (half an hour).
```
Do we have a reason to state along with it, it will  further clarify the statement.